src/symbolic_indexing.jl

```diff
@@ -192,6 +192,13 @@ function SymbolicIndexingInterface.is_observed(sys::PartitionedGraphSystem, sym)
     haskey(sys.compu_namemap, sym)
 end
 
+function SymbolicIndexingInterface.observed(sys::PartitionedGraphSystem, syms::Union{Vector{Symbol}, Tuple{Vararg{Symbol}}})
+    fs = map(syms) do sym
+        observed(sys, sym)
+    end
+    obs = map(f -> f(u, p, t), fs)
```

Comment:
```suggestion
    map(syms) do sym
        observed(sys, sym)(u, p, t)
    end
```
those two map operations can be fused into one actually